visualization/app/codeCharta/ui/codeMap/codeMap.label.service.spec.ts

```diff
@@ -47,27 +47,27 @@ describe("CodeMapLabelService", () => {
 	}
 
 	function withMockedThreeCameraService() {
-		threeCameraService = jest.fn<ThreeCameraService>(() => {
-			return {
-				camera: {
-					position: {
-						distanceTo: jest.fn()
-					}
+		threeCameraService = Object.assign(threeCameraService, {
```

Comment:
```suggestion
		Object.assign(threeCameraService, {
```

Would it not work to just assign camera as in: `threeCameraService.camera = { position { distanceTo: jestFn() } }`?

Reply:
Works, yes